pom.xml

```diff
@@ -27,6 +27,10 @@
             <id>jitpack.io</id>
             <url>https://jitpack.io</url>
         </repository>
+        <repository>
+            <id>central</id>
+            <url>https://repo1.maven.org/maven2/</url>
+        </repository>
```

Comment:
What's this for?

Reply:
I couldn't get `com.fasterxml.jackson.core.jackson-databind` to satisfy without it.

Reply:
Weird. It just comes down from maven central automatically on my machine. And in Github actions...

Maybe it's a weird versioning thing. Probably fine to explicitly include it here